packages/native-ui/src/config/theme/helpers/Button.ts

```diff
@@ -8,50 +8,53 @@ export const invertedOutlineScheme = (scheme: Scheme) => ({
   colorScheme: scheme,
   variant: 'outline',
   value: {
+    borderColor: `$${scheme}400`,
+
     _text: {
-      color: `$${scheme}50`,
+      color: `$${scheme}100`,
     },
 
     _icon: {
-      color: `$${scheme}50`,
+      color: `$${scheme}100`,
     },
 
     _spinner: {
       _svg: {
-        color: `$${scheme}50`,
+        color: `$${scheme}100`,
       },
     },
 
     ':active': {
       backgroundColor: `$${scheme}900`,
+      borderColor: `$${scheme}400`,
```

Comment:
not sure if this applies to inverted only, 
on `inverted` false (light mode only) the border colour doesn't change on pressed (active) so in the case of cyan it stays at 400 and it the other `colorScheme stays` at 500 
<img width="1072" alt="Screenshot 2024-08-08 at 16 25 20" src="https://github.com/user-attachments/assets/a78dff0b-82ca-4115-8d92-08daf48db993">

Reply:
Yes this is inverted only

Reply:
The normal outline did need updating though, good spot